Dockerfile

```diff
@@ -1,33 +1,13 @@
-FROM node:alpine3.15 as react
-
-WORKDIR /react
-
-COPY ./react/package* ./
-
-# For fix noticy about update npm. If you want to fix that, delete '#' in start next line.
-#RUN npm install -g npm@8.15.0
-
-RUN npm install
-
-COPY ./react .
-RUN npm run build
-
 FROM node:alpine3.15 as angular

```

Comment:
`RUN npm CMD` should not be in a dockerfile, it should be in the github actions YML.
Your dockerfile should just prepare and start the  webserver. Your github action should compile angular into static before the build step, that would be handled by the nginx running inside the container later.

Reply:
Fixed